main.py

```diff
@@ -12,7 +13,7 @@ def _configure_logging(configuration):
     dictConfig(configuration['logging'])
 
 
-def main():
+if __name__ == '__main__':
```

Comment:
I know you mentioned that having a `main()` and calling it from `if __name__ == '__main__'` did not work. Why was that again?

Reply:
It had something to do with multiprocessing, but I just moved multiprocessing into analyze so we can revert back to having `main()`